### ISSUE TITLE
Cite this repo on the Apache docs page, please!

### ISSUE THREAD

**Issue by tbrowder** [icing/mod_md#290]
I finally found this page (thanks to a reader) after asking for help at Let’s Encrypt.

Great module!

**Comment by tbrowder** [comment]
On Mon, Jun 13, 2022 at 02:49 Stefan Eissing ***@***.***>
wrote:

> Thank you! Glad you enjoy it.
>
I remember when its status was first announced, and I was estatic when it
was first released and put it to use at once. And I haven't looked back.
Starting was fairly easy, but recently I ran into trouble when I started
moving vhosts to different physical servers--that's when the tutorial would
have really been helpful.

> Where would you expect to find the information on the apache httpd site
> and what would it look like?
>
Hi, Stefan, thanks for replying so quickly. I would like to see it first on
the top of the docs section, say in the How To / Tutorials. Also a link at
the top of the mod_md page in the introduction would be great.

Blessings,

-Tom
